client/state/plugins/installed/selectors-ts.ts

```diff
@@ -228,6 +228,15 @@ export const getFilteredAndSortedPlugins = createSelector(
 	}
 );
 
+export function getPluginsWithUpdates( state: AppState, siteIds: number[] ) {
+	return getFilteredAndSortedPlugins( state, siteIds, undefined )
+		.filter( _filters.updates )
```

Comment:
Is there a reason why we're passing `undefined` as the third arg of `getFilteredAndSortedPlugins` when we can use it to pass a `pluginFilter` and use `updates`?

Reply:
No reason, probably just how it was before. I switched it.